### ISSUE TITLE
Rays do not turn off when hand are facing you

### ISSUE THREAD

**Issue by wiwei** [microsoft/MixedRealityToolkit-Unity#4377]
Ported to Github, reference 20978095

In the shell, hand rays turn off when hands are facing you. This is encoded in MRTK via the pointer IsInPointingPose bool. Hand rays are currently not turning off when the hands are facing you.

**Comment by keveleigh** [comment]
I believe this was implemented (by @julenka?), unless it regressed recently.

**Comment by julenka** [comment]
Yes I think this regressed a while back actually.

**Comment by lukastoenneMS** [comment]
Do you have any idea when this regressed and/or where i can find the original code? Without further information i am not sure what the expected behavior is.

**Comment by julenka** [comment]
Removing GA blocker because it's not issue in UX control functionality. Will fix after GA.

**Comment by julenka** [comment]
> Do you have any idea when this regressed and/or where i can find the original code? Without further information i am not sure what the expected behavior is.

WMRArticulatedHand has field IsInPointingPose, as does IMixedRealityController. LinePointer has check:

```
 public override bool IsInteractionEnabled =>
                // If IsTracked is not true, then we don't have position data yet (or have stale data),
                // so remain disabled until we know where to appear (not just at the origin).
                IsFocusLocked || (IsTracked && Controller.IsInPointingPose && base.IsInteractionEnabled);
```

It looks like the logic for determining IsInPointingPose is either broken.

**Comment by lukastoenneMS** [comment]
This appears to be a by-design feature of the LinePointer/ShellHandRayPointer. They have two line renderers, one for the full line and one for the stippled line. The full line (as well as interactions of the pointer) gets disabled correctly when tilting the hand, but the stippled renderer is not disabled. Note also that any focus or other actual pointer behavior gets disabled, this appears to be a purely visual artifact, i'm going to give this back to whoever wrote the code, since it's really convoluted and hard to follow.

**Comment by julenka** [comment]
> This appears to be a by-design feature of the LinePointer/ShellHandRayPointer. They have two line renderers, one for the full line and one for the stippled line. The full line (as well as interactions of the pointer) gets disabled correctly when tilting the hand, but the stippled renderer is not disabled. Note also that any focus or other actual pointer behavior gets disabled, this appears to be a purely visual artifact, i'm going to give this back to whoever wrote the code, since it's really convoluted and hard to follow.

@railboy, could you please take a look? It looks like the stippled line renderer needs to turn off when the hands are not in the 'pointing pose'.

**Comment by Troy-Ferrell** [comment]
@julenka can this be closed?

**Comment by Troy-Ferrell** [comment]
Confirmed fixed in editor
